packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1230,13 +1220,39 @@ export class StripeHelper {
             )
           );
         }
-        plan = await this.expandResource(subscription.plan, PLAN_RESOURCE);
-        abbrevProduct = await this.expandAbbrevProductForPlan(plan);
-        break;
+
+        /** @type {Plan | undefined | null} */
+        const plan = await this.expandResource(
+          subscription.plan,
+          PLAN_RESOURCE
+        );
+        /** @type {AbbrevProduct | undefined} */
```

Comment:
Shouldn't be needed, this is TypeScript.

Reply:
True.  These comments were moved along with their code.  I'll remove them.